File: 11-2024/Python/axbxc.py
# Cho một số K. Hãy tìm xem có bao nhiêu bộ số (A, B, C) mà A*B*C<=K. Các bộ khác chỉ khác nhau về thứ tự số vẫn được chấp nhận.
k = int(input())

ans = 0
for i in range(1, k + 1):
        for j in range(1, k // i + 1):
                ans += k // (i * j)

print(ans)

File: 11-2024/CPP/axbxc.cpp
// Cho một số K. Hãy tìm xem có bao nhiêu bộ số (A, B, C) mà A*B*C<=K. Các bộ khác chỉ khác nhau về thứ tự số vẫn được chấp nhận.
#include <bits/stdc++.h>
using namespace std;

int k;
long long ans;

int main()
{
        ios_base::sync_with_stdio(0);
        cin >> k;
        for (int i = 1; i <= k; ++i)
                for (int j = 1; j <= k / i; ++j)
                        ans += k / (i * j);
        cout << ans;
}